ExampleMod/Common/Commands/ItemCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Terraria;
+using Terraria.ModLoader;
+
+#nullable enable
```

Comment:
Appears to be unnecessary? Doesn't matter much, but could confuse people unfamiliar with nullables.